plugin/trino-phoenix5/src/test/java/io/trino/plugin/phoenix5/TestPhoenixConnectorTest.java

```diff
@@ -227,12 +227,12 @@ public void testAddAndDropColumnName()
             if (columnName.equals("an'apostrophe")) {
                 assertThatThrownBy(() -> testAddAndDropColumnName(columnName, requiresDelimiting(columnName)))
                         .hasMessageContaining("Syntax error. Mismatched input");
-                abort("TODO");
```

Comment:
retain "TODO"

Reply:
TODO comment already exists in this test:
```
            // TODO: Investigate why these two case fail
```